ecasbot/__init__.py

```diff
@@ -429,8 +429,20 @@ def handle_msg(message) -> None:
             """
             try:
                 # Removing messages from restricted members...
-                if self.__check_message_entities(message) or self.__check_message_forward(
-                        message) or self.__check_message_spam(message):
+                entities = self.__check_message_entities(message)
+                forward = self.__check_message_forward(message)
+                spam = self.__check_message_spam(message)
+                author_name = "{} - {} {}".format(message.from_user.id,
+                                                  message.from_user.first_name, message.from_user.last_name)
+
+                self.__logger.debug("chat: {} ({}) ".format(message.chat.id, message.chat.title))
```

Comment:
Also this need to be combined to a single row.

Reply:
Messages can be pretty long, so I'd prefer to keep those split into several lines.

There is no delimiter between messages, I wonder if I should add smth like '----' between those